README.md

```diff
@@ -12,69 +12,73 @@ This specification defines the additional OpenID Connect authentication response
 - Adam Lemmon (Trybe.ID)
 - Tobias Looker (Mattr)
 
+## Terminology
+
+If the specification states "Verifiable Presentation" this also includes "Verifiable Credentials". 
 ## Introduction
 
 Notes:
 - this should really start with the explanation of the role signatures play in VCs (can be both JSON or JSON-LD) and that there are two widely used proof types (JWTs and LD-proofs)
 - Explain why is there a need for this extension?
 
 ## Overview
-- RP requests `vp_token` by adding an additional destination `vp_token` in the `claims` request parameter. 
-- Authentication event information is conveyed via the id token while it's up to the RP to determine what (additional) claims are allocated to id_token and vp_token, respectively.
-- `vp_token` is provided in the same response as the `id_token`. Depending on the response type, this can be either the authentication response or the token response. 
-- If the vp_token is returned in the frontchannel, a hash (`vp_hash`) of `vp_token` must be included in `id_token`.
+
+This specifications introduces two mechanisms to provide Verifiable Presentations:
+
+* ID Token as Verififiable Presentation: An ID Token may contain a claim `vp` or `vc` as defined in [JWT proof format](https://www.w3.org/TR/vc-data-model/#json-web-token), i.e. it is a valid OpenID Connect ID Token and a Verifiable Presentation at the same time. Consequently, this mechanism utilizes (and supports) the JWT proof format only. 
+* dedicated VP Token: a Verifiable Presentation is provided in a separate artifact designated as VP Token. Such a token is provided to the RP in addition to an `id_token` in the `vp_token` parameter. VP Tokens support Verifiable Presentations in JSON-LD as well JWT format including all respective proof formats. The `vp_token` is provided in the same response as the `id_token`. Depending on the response type, this can be either the authentication response or the token response. Authentication event information is conveyed via the id token while it's up to the RP to determine what (additional) claims are allocated to id_token and vp_token, respectively. If the vp_token is returned in the frontchannel, a hash (`vp_hash`) of `vp_token` must be included in `id_token`.
```

Comment:
If `vp_token` and `id_token` and both requested, is `claims` required?
I am thinking as an OP how to determine which claims to put into the `vp_token` and which to put into the `id_token`.

Does this: `it's up to the RP to determine what (additional) claims are allocated to id_token and vp_token` imply something similar?

Reply:
yes the claims parameter. I am thinking in order to explicitly determine which claims to be in which artifacts, `id_token` vs. `vp_token`?

Reply:
that’s what the claims parameter is used for. Claims that shall go into the vp_tokens are requested underneath „vp_token“ in the claims parameter.

Reply:
Right right! I'm just mentioning should we make the presence of `claims` a MUST if both `id_token` and `vp_token` are present?

Reply:
The presence of the claims parameter in the request is a pre-requisite for the presence of the vp_token in the response. Can you propose text?

Reply:
I don't mind your language there actually, but last question before landing on some language I think would be if we allow just `vp_token` as a response_type? and if so would we also require `claims` be used in that case?  Or we only require `claims` if both `vp_token` and `id_token` are present?

Reply:
I suggest to postpone this discussion until we have discussed #16